### PR TITLE
Mobile version Final PR

### PR DESCRIPTION
**Pull request by misikirdelelew**
In this PR i have developed 

- The work section of the mobile version
- The about section of the mobile version
- The contact form of the mobile version based on the Figma design and microverse requirement

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -85,3 +85,334 @@ header {
...
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.card-sub-container {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
```
Comment (levy002) [inline css/style.css:131]: - [ ] Kindly make sure to change the `Canopy` color in order to meet the Figma design. :wink: 

     - ### YOUR DESIGN
     
    image

     - ### FIGMA DESIGN
     
    image

File: index.html [hunk 2]
```diff
@@ -42,6 +42,219 @@ <h2 class="connect-link">LET’S CONNECT</h2>
...
+                    <h2 class="heading-title">
+                        Tonic
+                    </h2>
+                    <article class="card-sub-container">
+                        <h2 class="title-canopy">CANOPY</h2>
+                        <img src="" alt="">
+                        <h2 class="title-developer">Back End Dev</h2>
+                        <img src="" alt="">
+                        <h2 class="title-year">2015</h2>
+                    </article>
```
Comment (levy002) [inline index.html:95]: - [ ] Kindly make sure to add the dots which are acting as the separators between these contents.

   - ### YOUR DESIGN
  
       image

   - ### FIGMA DESIGN
   
      image

File: css/style.css [hunk 3]
```diff
@@ -85,3 +85,334 @@ header {
...
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  background-color: #fff;
+  width: 327px;
+  height: 101px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 12px;
+}
```
Comment (levy002) [inline css/style.css]: - [ ] Kindly make sure your `About Myself` title have the same color as the one on the Figma design :wink: 

  - ### YOUR DESIGN
  
    image

  - ### FIGMA DESIGN
  
   image

File: css/style.css [hunk 4]
```diff
@@ -85,3 +85,334 @@ header {
...
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media {
+  display: flex;
+  list-style-type: none;
+  gap: 19px;
+}
```
Comment (levy002) [inline css/style.css]: - [ ] Kindly make sure to center the social media links in the container in order to look like the provided Figma design :wink: 

   - ### YOUR DESIGN
   
     image

   - ### FIGMA DESIGN
   
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @MishDM   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -42,6 +42,223 @@ <h2 class="connect-link">LET’S CONNECT</h2>
...
+                <form action="https://formspree.io/f/xpzcccob" method="POST" class="contact-form" id="contact-form">
+                    <label for="name"></label>
+                    <input type="text" name="Name" placeholder="Write name." required maxlength="30" id="name">
+                    <label for="email"></label>
+                    <input type="email" name="Email" placeholder="example@gmail.com" required id="email">
+                    <label for="message-container"></label>
+                    <textarea name="message" id="message-box" cols="30" rows="10"
+                        placeholder="write your message in here" maxlength="500" required></textarea>
+                    <button type="submit" class="get-in-touch-btn
+                    ">Get It Touch</button>
```
Comment (NduatiKagiri) [inline index.html:254]: - [ ] Please make sure that your submit button is aligned together / is in line with the form.
-   Your design
     portfo2

-   Figma design
     portfo3

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @MishDM ,

# CHANGES REQUIRED ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

 ## Highlights ✔️ 
✅ GitHub Flow observed
✅ No linters errors


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @MishDM  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🏴 
 
-  [ ] Linters are passing 💯 
-  [ ] Right commit messages 🍸 
-  [ ] Proper Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**